example_template.md

```diff
@@ -0,0 +1,102 @@
+<!-- DON'T EDIT THIS SECTION, INSTEAD RE-RUN doctoc TO UPDATE -->
```

Comment:
Opinion: Let's add the title above the table of contents!

Reply:
Opinion: Lets remove the table of contents

Reply:
I'm not sure about removing it, I think it's a clear way to show what will be covered